.vsts-pipelines/variables/docker-images.yml

```
@@ -1,4 +1,7 @@
 variables:
   imageNames.imageBuilder.linux: mcr.microsoft.com/dotnet-buildtools/image-builder:debian-20190305214722
   imageNames.imageBuilder.windows: mcr.microsoft.com/dotnet-buildtools/image-builder:nanoserver-20190305134632
+  imageNames.imageBuilder.withrepo: microsoft-dotnet-imagebuilder-withrepo:$(Build.BuildId)-$(System.JobId)
```

Thoughts on dropping `microsoft-dotnet-` from these image names to keep them shorter?  It would help reading the build output.
```suggestion
  imageNames.imageBuilder.withrepo: imagebuilder-withrepo:$(Build.BuildId)-$(System.JobId)
```

---

Yeah, that makes sense.